libs/chat-workflow/src/main/java/org/finos/springbot/workflow/response/templating/TeamsWorkResponse.java

```diff
@@ -0,0 +1,32 @@
+package org.finos.springbot.workflow.response.templating;
+
+import org.finos.springbot.workflow.annotations.WorkMode;
+import org.finos.springbot.workflow.content.Addressable;
+import org.finos.springbot.workflow.form.ButtonList;
+import org.finos.springbot.workflow.form.ErrorMap;
+import org.finos.springbot.workflow.response.WorkResponse;
+
+import java.util.Map;
+
+public class TeamsWorkResponse extends WorkResponse {
+    private final String summary;
```

Comment:
You shouldn't need to add a field to the TeamsWorkResponse - instead, just declare the name of a key and then add it to the `data` map.

Reply:
Also, this is breaking the separation of concerns in spring-bot, as you've added a Teams-Specific class  to chat-workflow.  Everything teams-related should go in the teams project.  

I think you'll see based on what I've written already that this class is unnecessary

Reply:
Yes, right.. we will move to team module